src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -914,14 +1175,16 @@ retrieve_handles_from_fuse(int idx)
 		goto err;
 	}
 
-	rc = d_hash_table_create(D_HASH_FT_EPHEMERAL | D_HASH_FT_MUTEX | D_HASH_FT_LRU, 6, NULL,
-				 &hdl_hash_ops, &dfs_list[idx].dfs_dir_hash);
-	if (rc) {
+	rc = dcache_init(&dfs_list[idx]);
```

Comment:
I think we should add an option (env variable) to disable the dcache entirely (not the default, but just an option).

Reply:
- [ ] Add configuration variable allowing to disable cache

Reply:
Fixed with commit b530789d44f8fded639bcf685b2e3621bf9b24b0